src/components/products/FilterAndSort.jsx

```diff
@@ -23,7 +23,7 @@ const FilterAndSort = () => {
             type="radio"
             name="sorting"
             checked={filterState.sorting === "lth"}
-            onClick={() =>
+            onChange={() =>
               filterDispatch({ type: "LOW_TO_HIGH", payload: "lth" })
```

Comment:
Suggestion: this case can be combined as {type:"SORT_BY", payload:"LOW_TO_HIGH"}. I mean instead of having 
 two diff cases for high_to_low and low_to_high, you can have it under one roof(sort_by)

Reply:
Thanks for the suggestion. I've noted it down for next round of refactoring.